spring-di-3/pom.xml

```diff
@@ -42,11 +42,33 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-test</artifactId>
         </dependency>
```

Comment:
Shouldn't these just come as transitive dependencies from spring-boot-dependencies?

Reply:
@anastasiosioannidis i think this is ok, the spring-boot-dependencies are declared in the dependency Manangement tag in the parent-spring-6, so inherited modules will have to pull the dependency they want.